CSS/CSS.sublime-syntax

```diff
@@ -7,6 +7,14 @@ file_extensions:
   - css.erb
 scope: source.css
 variables:
+  # Many variable names taken directly from https://www.w3.org/TR/CSS22/grammar.html#scanner
+  h: '[0-9a-fA-F]'
+  unicode: '\\{{h}}{1,6}(?:\r\n|[ \t\r\n\f])?'
+  escape: '(?:{{unicode}}|\\[^\r\n\f0-9a-fA-F])'
+  nmstart: '(?:[_a-zA-Z]|[^[:ascii:]]|{{escape}})'
+  nmchar: '(?:[_a-zA-Z0-9-]|[^[:ascii:]]|{{escape}})'
+  ident: '-?{{nmstart}}{{nmchar}}*'
+
```

Comment:
In the case that it is desired to support such characters, unicode property escapes should probably be used: http://www.regular-expressions.info/unicode.html#category.

My guess is something like `[\p{L}\p{M}\p{S}\p{N}_&&\P{Nd}]`. That will be letters, marks, symbols, numbers and `_`, but minus decimal digit numbers (`\P{Nd}`).